nixos/modules/services/networking/quassel.nix

```diff
@@ -62,13 +64,15 @@ in
 
       dataDir = mkOption {
         default = "/home/${user}/.config/quassel-irc.org";
+        type = types.str;
```

Comment:
Shouldn't this be types.path?

Reply:
Im not sure what is containted in the .config file, there could be secrets in there

Reply:
No, since this location does not necessarily exist at eval-time.